components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -193,7 +207,9 @@ class PromptFeature private constructor(
         loginExceptionStorage = loginExceptionStorage,
         onNeedToRequestPermissions = onNeedToRequestPermissions,
         loginPickerView = loginPickerView,
-        onManageLogins = onManageLogins
+        onManageLogins = onManageLogins,
+        creditCardPickerView = creditCardPickerView,
+        onManageCreditCards = onManageCreditCards
     )
 
     @Deprecated("Pass only activity or fragment instead")
```

Comment:
Outside of this review:

@Amejia481 should this constructor be removed? It would make a good first bug, if so.

Reply:
Sure, I will create a ticket for it.

Reply:
Associated ticket : #11387